tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -23,9 +23,10 @@ ${SPAWNER_LINK}=        xpath=//*[text()="Launch Jupyter"]
 Open Data Science Projects Home Page
     [Documentation]    Verifies submenu Settings > "Data Science Projects" is visible
     # Page Should Contain    Data Science Projects
-    Click Link      Data Science Projects
+    SeleniumLibrary.Wait Until Page Contains    Data Science Projects
```

Comment:
can you remove the Lib name?

Reply:
wait do you need to wait for this? in theory, if you run `Wait for RHODS Dashboard to Load` before calling `Open Data Science Projects Home Page` you should not get any failures

Reply:
Without the Wait it would fail while the page was still loading.
>can you remove the Lib name?
I prefer to keep it in, as without the name it's always marked as an issue in my IDE. I also think it's better this way, so that it doesn't fail when called from somewhere that didn't set the search order.

Reply:
>Without the Wait it would fail while the page was still loading.


probably you need to call `Wait for RHODS Dashboard to Load` it before running `Open Data Science Projects Home Page`

Reply:
To me it makes more sense to keep the wait inside this keyword. If the link that the keyword has to click is already present it won't add any run time to the tests, otherwise it will prevent a failure that could've been avoided.